plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -210,7 +210,7 @@ export default class ChannelHeaderPlug extends React.PureComponent<ChannelHeader
 
         if (components.length === 0) {
             return null;
-        } else if (components.length <= 5) {
+        } else if (components.length <= 15) {
```

Comment:
Isn't 15 a bit high? I imagine the header would be pretty squished at this point. What about 8 or 9?

Reply:
I went high because if someone installs that many, they intend to use them. Why hide it from them? I figure just do it on an absolutely ridiculous number of buttons.

Reply:
Fair enough